lib/simplifyExpression/basicsSearch/simplifyDoubleUnaryMinus.js

```diff
@@ -2,36 +2,28 @@ const clone = require('../../util/clone');
 
 const ChangeTypes = require('../../ChangeTypes');
 const Node = require('../../node');
+const {build, query} = require('math-nodes');
 
 // Simplifies two unary minuses in a row by removing both of them.
 // e.g. -(- 4) --> 4
 function simplifyDoubleUnaryMinus(node) {
-  if (!Node.Type.isUnaryMinus(node)) {
```

Comment:
One way to speed up porting is to change the implementation of `Node.Type.isUnaryMinus` to call `query.isNeg` and do the same for the rest of the functions in `Node.Type`.